day05Yatin/create_s3.sh

```
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euo pipefail
+
+<< TASK
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)